down-the-stack-book/README.md

```diff
@@ -0,0 +1,11 @@
+# Down the Stack Slides
+
+To read as an mdbook:
+
+1. `cargo install mdbook`
+2. `mdbook serve`
+
```

Comment:
The mdbook does not show the diagrams, do I need to install something else to show them?

Reply:
`cargo install mdbook-mermaid`